src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HomeHealthCertifiedPersonTableViewCell.swift

```diff
@@ -39,6 +39,8 @@ class HomeHealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierPr
 		iconView.image = cellModel.iconImage
 		backgroundImageView.image = cellModel.backgroundImage
 		backgroundGradientView.type = cellModel.backgroundGradientType
+
+		accessibilityIdentifier = cellModel.accessibilityIdentifier
```

Comment:
Would be nice to move the accessibilityIdentifier from line 17 also to the model

Reply:
done